code/modules/shuttle/shuttle_rotate.dm

```diff
@@ -105,10 +101,6 @@ If ever any of these procs are useful for non-shuttles, rename it to proc/rotate
 	params = NONE
 	return ..()
 
-/obj/machinery/door/airlock/survival_pod/shuttleRotate(rotation, params)
```

Comment:
What was these about ?

Reply:
Apparently the survival pod airlock would not rotate when it was inside a shuttle that rotated, the dirs on survival pod airlocks were all messed up, there used to be 2 survival pod airlocks a horizontal and vertical variant but all the dirs were put into a single icon a while ago and made into a single airlock
After removing the shuttleRotate stuff I tested it with the nuke ops ship and nav computer, both the airlock and assembly have unique icons for all 4 dirs and they rotate as you would expect them to